maas/plugins/galera_check.py

```diff
@@ -74,8 +74,8 @@ def print_metrics(replica_status):
            replica_status['wsrep_commit_window'], 'sequence_delta')
     metric('wsrep_cluster_size', 'int64',
            replica_status['wsrep_cluster_size'], 'nodes')
-    metric('queries_per_second', 'int64',
-           replica_status['Queries'], 'qps')
+    metric('queries', 'int64',
```

Comment:
Removing this metric might affect some of the people who use our MaaS metrics. @mancdaz do we usually straight off just rename something or do we report two metrics for backwards compatibility?